detekt-rules-errorprone/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/bugs/ElseCaseInsteadOfExhaustiveWhen.kt

```diff
@@ -70,6 +79,10 @@ class ElseCaseInsteadOfExhaustiveWhen(config: Config = Config.empty) : Rule(conf
         if (whenExpression.elseExpression == null) return
 
         val subjectType = subjectExpression.getType(bindingContext)
+        if (ignoredSubjectTypes.contains(subjectType?.fqNameOrNull().toString())) {
```

Comment:
nit: 
```suggestion
        if (ignoredSubjectTypes.contains(subjectType?.fqNameOrNull()?.toString())) {
```

I don't know what `contains` thinks about `null`. But checking for `"null"` doesn't seem 100% correct.